pandas/compat/__init__.py

```diff
@@ -20,14 +19,6 @@
 IS64 = sys.maxsize > 2 ** 32
 
 
-# ----------------------------------------------------------------------------
-# functions largely based / taken from the six module
-
-# Much of the code in this module comes from Benjamin Peterson's six library.
-# The license for this library can be found in LICENSES/SIX and the code can be
-# found at https://bitbucket.org/gutworth/six
-
-
```

Comment:
The LICENSES folder doesn't contain SIX. So I guess this comment is not valid anymore.